src/fields/mod.rs

```diff
@@ -0,0 +1,353 @@
+//! Various prime order fields. Each is defined in its own submodule because otherwise the names of
+//! items generated by the [`ff::PrimeField`] macro collide.
```

Comment:
Update this doccomment to remove mention of `ff::PrimeField`. Also, is it necessary to make distinct submodules anymore? It looks like all the generated items start with e.g. `fiat_p521`. Then again I don't necessarily mind having a module per field.